src/Binovo/ElkarBackupBundle/Command/RemoteRestoreCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Binovo\ElkarBackupBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Binovo\ElkarBackupBundle\Lib\LoggingCommand;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+
+
+class RemoteRestoreCommand extends ContainerAwareCommand
+{
+    const PARAM_URL = 'url';
+    const PARAM_SOURCE_PATH = 'sourcePath';
+    const PARAM_REMOTE_PATH = 'remotePath';
+    const PARAM_SSHARGS = 'sshArgs';
+
+    protected function configure()
+    {
+        $this->setName('elkarbackup:restore_backup');
+        $this->setDescription('Restore a backup into remote machine.');
+        $this->addArgument(self::PARAM_URL, InputArgument::REQUIRED,'The connection URL for the remote machine');
+        $this->addArgument(self::PARAM_SOURCE_PATH, InputArgument::REQUIRED,'The source path to the directory or files that should be copied');
+        $this->addArgument(self::PARAM_REMOTE_PATH, InputArgument::REQUIRED,'The remote path on which the directory or files should be retore');
+        $this->addArgument(self::PARAM_SSHARGS,InputArgument::OPTIONAL,'The remote machine extra ssh parameters');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $container = $this->getContainer();
+        $manager = $container->get('doctrine')->getManager();
+        
+        $logger = $this->getContainer()->get('BnvWebLogger');
+        $translator = $this->getContainer()->get('translator');
+        $context = array('source' => 'RestoreBackups');
+                
+        $volumes = $manager->getRepository('BinovoElkarBackupBundle:BackupLocation'); 
+
+        $url = $input->getArgument(self::PARAM_URL);
+        $sourcePath = $input->getArgument(self::PARAM_SOURCE_PATH);
+        $remotePath = $input->getArgument(self::PARAM_REMOTE_PATH);
+        $sshArgs = $input->getArgument(self::PARAM_SSHARGS);
+        
+        if($sshArgs !== ''){
+          $port = preg_replace('/[^0-9]/', '', $sshArgs);
+            if ($port !== ''){
+              $sshArgs = '-p '.$port;
+            } else {
+              $sshArgs = '-p 22';
+            }
+          $logger->info('We ssh have arguments, take care of it!'.$sshArgs,$context);
```

Comment:
Not really sure what is this info log for, doesn't seem properly redacted.

Reply:
We don't need this log but sometimes we can introduce more parameters than we take care of it in sshargs and we can have problems with them, that's why I think we should register this information, maybe with a different message, and you are right! is not properly redacted.

SSH arguments used for this restore task?